examples/POD/lorenz.jl

```diff
@@ -0,0 +1,40 @@
+using ModelOrderReduction
+using Plots
+using OrdinaryDiffEq
+
+function lorenz_prob()
+    function lorenz!(du,u,p,t)
+        du[1] = p[1]*(u[2]-u[1])
+        du[2] = u[1]*(p[2]-u[3]) - u[2]
+        du[3] = u[1]*u[2] - p[3]*u[3]
+    end
+
+    u0 = [1,0,0]
+    p = [10,28,8/3]
+    tspan = (0,100)
+    prob = ODEProblem(lorenz!,u0,tspan,p)
+    sol = solve(prob,Tsit5())
+    sol
+end
+
+sol = lorenz_prob()
+solution = Matrix(reduce(hcat,sol.u)')
```

Comment:
Array(sol) is what you want to be using. You are not "correctly" (to be understood in the usual context of POD) concatenating the data snapshots (see comment on matricize). For a test this is alright but this isn't really how PCA is used in the context of model order reduction.